Example/ImageKit/FetchImageViewController.swift

```diff
@@ -39,28 +39,22 @@ class FetchImageViewController: UIViewController {
     }
     
     @IBAction func OnClickTransformation3(_ sender: Any) {
-        let url = ImageKit.shared.url(src: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg", transformationPosition: TransformationPosition.PATH)
-            .overlayImage(overlayImage: "logo-white_SJwqB4Nfe.png")
-            .overlayX(overlayX: 10)
-            .overlayY(overlayY: 20)
-            .create()
+        guard let url = try? ImageKit.shared.url(src: "https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg", transformationPosition: TransformationPosition.PATH)
+            .setResponsive(view: imageView)
+            .create() else { return }
```

Comment:
This should include the overlay in this example, using layer syntax as it was originally

output url should be
https://ik.imagekit.io/demo/tr:l-image,i-logo-white_SJwqB4Nfe.png,ox-10,l-end/medium_cafe_B1iTdD0C.jpg

Reply:
Done